server/src/util/logger.ts

```diff
@@ -0,0 +1,15 @@
+import winston from 'winston'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/c0dejohn/e-commerce-backend/pullRequest?prid=8292833)
```suggestion
import winston from "winston"
```